src/adapters/adgear.js

```diff
@@ -0,0 +1,130 @@
+var CONSTANTS = require('../constants.json');
+var utils = require('../utils.js');
+var bidmanager = require('../bidmanager.js');
+var bidfactory = require('../bidfactory.js');
+var adloader = require('../adloader.js');
+var Adapter = require('./adapter.js');
+
+var AdGearAdapter = function AdGearAdapter() {
+  var proto = ('http:' === document.location.protocol) ? 'http' : 'https';
```

Comment:
We are requiring all adapters to hardcode to `HTTPS` now. Please remove the `HTTP` option.